<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaoZuo_Apply</class>
 <widget class="QDialog" name="LaoZuo_Apply">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>目前可報名的勞作活動</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 252, 235);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_bShowOther">
      <property name="text">
       <string>顯示其他時段</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_bShowMidd">
      <property name="text">
       <string>顯示中午時段</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_bShowPM">
      <property name="text">
       <string>顯示下午時段</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_bFilterFull">
      <property name="text">
       <string>篩掉已滿</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>851</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;微軟正黑體&quot;;
background-color: rgb(231, 255, 241);</string>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnReserve">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[test]LaborMiner</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
